sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpMessageConverter.cs

```diff
@@ -53,7 +53,7 @@ public static AmqpMessage BatchSBMessagesAsAmqpMessage(IReadOnlyCollection<Servi
         ///
         /// <returns>The batch <see cref="AmqpMessage" /> containing the source messages.</returns>
         ///
-        private static AmqpMessage BuildAmqpBatchFromMessage(IReadOnlyCollection<ServiceBusMessage> source, bool forceBatch)
+        public virtual AmqpMessage BuildAmqpBatchFromMessage(IReadOnlyCollection<ServiceBusMessage> source, bool forceBatch)
```

Comment:
I don't think we need an explicit `firstMessage` reference - we don't seem to use it.

Reply:
Does this need to be public?

Reply:
Same question here about whether this needs to be public.

Reply:
No I don't think this does, switched it to private.